text/trace/0107-sampling-priority.md

```diff
@@ -0,0 +1,81 @@
+# Associating sampling priority with the trace using tracestate
+
+Propagating `sampling.priority` (calculated by probability sampler) in a
+`tracestate` will enable a better experience for system with the independent
+stateless configuration of components sampling.
+
+## Motivation
+
+Consistent sampling decision made in each app of a distributed trace is
+important for better user experience of trace analysis. Consistency is achieved
+by following means:
+- Same hashing algorithms used across all apps in a trace.
+- Sampling flag propagated from the head component/app is used by downstream apps to sample in a given trace.
+
+Increasingly more apps participate in distributed tracing. With the 
+standardized wire format for context propagation, there is bigger chance that 
+algorithm for sampling chosen by one app may not match another app. 
+Since coordination of sampling algorithms across many apps not always possible, 
+OpenTelemetry must provide a way to share sampling hints between apps.
+
+Beyond sampling hints infrastructure, it is beneficial for the community to
+agree on standard behavior and hints exposed by probability sampler.
+
+The `sampling.priority` property is used by many vendors for various purposes.
```

Comment:
Did you mean `sampling.probability`?  `sampling.priority` is a standard OpenTracing span tag that can be used to influence/override sampling decisions. In the context of this OTEP "priority" doesn't seem to make sense.

Reply:
In the proposal the intent is similar to sampling.priority in OpenTracing. So if the trace is sampled in by first component it passes the priority to next component to make sampling decision.

Reply:
I don't think it's similar. In OpenTracing the word "priority" is a hint from the user. Here it's an output from the sampler.